main.go

```diff
@@ -235,3 +244,19 @@ func kubeClientQPS() (float32, error) {
 	qps = float32(val)
 	return qps, err
 }
+
+// getCustomConfig returns a custom Kube-Linter configuration
+// it retrieves data from a ConfigMap if there is any valid
+func getCustomConfig(cfg *rest.Config) (*kubelinterCfg.Config, error) {
+	cmw, err := controller.NewConfigMapWatcher(cfg)
+	if err != nil {
+		return nil, fmt.Errorf("getting kube-linter checks configuration from configmap: %w", err)
+	}
+
+	klCfg, err := cmw.GetStaticKubelinterConfig(context.Background())
```

Comment:
I think we should ideally pass a context instance to the `getCustomConfig` and use it here. WDYT?

Reply:
I thought about it, but there was no context used on the upstream. I can send start it there and then send it as argument :+1:

Reply:
There's `if err := mgr.Start(signals.SetupSignalHandler()); err != nil {` in the main function, when `signals.SetupSignalHandler())` returns a context, so I think you can take it from there.